src/bindings/python/src/pyopenvino/core/executable_network.cpp

```diff
@@ -36,12 +36,8 @@ void regclass_ExecutableNetwork(py::module m) {
             // Update inputs if there are any
             Common::set_request_tensors(request, inputs);
             request.infer();
-
-            Containers::InferResults results;
-            for (const auto out : self.outputs()) {
-                results.push_back(request.get_tensor(out));
-            }
-            return results;
+            py::dict res = Common::outputs_to_dict(self.outputs(), request);
```

Comment:
you may also just `return Common::outputs_to_dict(self._outputs, self._request);`